src/lib/utils/sanitizeTiptapContent.ts

```diff
@@ -1,10 +1,6 @@
 import sanitizeHtml from 'sanitize-html';
 
-const allowedHosts = [
-	"c.clubsaur.us",
-	"clubsaur.us",
-	"localhost"
-]
+const allowedHosts = ['c.clubsaur.us', 'clubsaur.us', 'localhost'];
```

Comment:
only enable localhost on dev mode

Reply:
Fixed in [2d5e7ed](https://github.com/CoasterFan5/Clubsaurus/pull/150/commits/2d5e7edf1cb4c6c3eddd5a779cf62523a94d7443)